wiki/Client/Interface/Multiplayer/en.md

```diff
@@ -210,7 +211,9 @@ There are four different ways the winner of a match can be decided:
 
 ### Match setup buttons
 
-There are two large wide orange and blue buttons above the [chat console](/wiki/Client/Interface/Chat_console). The orange `Leave Match` button on the left is self-explanatory.
+There are two large wide orange and blue buttons above the [chat console](/wiki/Client/Interface/Chat_console).
+
+The orange `Leave Match` button on the left is self-explanatory. If the match host leave the match, the host will be automatically transferred to the remaining player in slot order. If there is no one in the match, it will automatically close, except for ones made by `!mp` commands, which will instead close after 30 minutes.
```

Comment:
```suggestion
The orange `Leave Match` button on the left is self-explanatory. If the match host leave the match, the host will be automatically transferred to the remaining player in slot order. If there is no one in the match, it will automatically close, except for ones made by [`!mp` commands](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands), which will instead close after 30 minutes.
```